connector.go

```diff
@@ -29,8 +28,13 @@ func (c *connector) Connect(ctx context.Context) (driver.Conn, error) {
 		schemaName = cli_service.TIdentifierPtr(cli_service.TIdentifier(c.cfg.Schema))
 	}
 
+	tclient, err := client.InitThriftClient(c.cfg)
```

Comment:
This is okay to fix the issue, but ideally I'd like to reuse the http client (not necessarily the thrift client). The http client has a connection pool and it is intended to be used across many go routines.